cosmos/profiles/base.py

```diff
@@ -180,8 +248,14 @@ def get_profile_file_contents(
             }
         }
 
+        if self.dbt_config_vars:
+            profile_contents["config"] = self.dbt_config_vars.as_dict()
+
         if self.disable_event_tracking:
-            profile_contents["config"] = {"send_anonymous_usage_stats": False}
+            logger.warning(
+                "disable_event_tracking is deprecated and will be removed in 2.0. You can use instead dbt_config_vars"
+            )
```

Comment:
This should be moved to the constructor and use the deprecation warning category, see an example here: https://github.com/astronomer/astronomer-cosmos/blob/1b7f4a06e06cb5af75bf411a28dca932ef37ab5d/cosmos/config.py#L69-L72
Would also suggest being explicit on how to get the same behavior by adding "send_anonymous_usage_stats=False"